lib/LaTeXML/Package/lxRDFa.sty.ltxml

```diff
@@ -18,11 +18,11 @@ use LaTeXML::Package;
 #======================================================================
 # Package Options
 DeclareOption('labels', sub {
-    Let(T_CS('\lxRDF@original@label'),           T_CS('\label'));
-    Let(T_CS('\lxRDF@original@longtable@label'), T_CS('\@longtable@label'));
+    Let(T_CS('\lxRDF@original@label'),             T_CS('\label'));
+    Let(T_CS('\lxRDF@originallx@longtable@label'), T_CS('\lx@longtable@label'));
```

Comment:
Maybe the inner `originallx` renaming here was accidental? That macro already started with `\lx`.

Reply:
yeah, weird renaming, but isn't actually wrong, so...